config/routes.rb

```diff
@@ -1,5 +1,4 @@
 Rails.application.routes.draw do
   mount DDS::Base, at: '/'
   get "apiexplorer" => "swaggerui#index"
-  root to: redirect("/portal", status: 302)
```

Comment:
we really need a root. Otherwise, if someone navigates to  https://apidev.dataservice.duke.edu, they get the default rails application